src/content/en/2019/javascript.md

```diff
@@ -289,24 +289,24 @@ Other top used JavaScript libraries include jQuery variants (jQuery migrate, jQu
 
 ### Frameworks and UI libraries
 
+<aside class="note">Although interesting, the results in this section are not accurate. As mentioned in our <a href="./methodology#wappalyzer">methodology</a>, the third-party detection library used in HTTP Archive (Wappalyzer) has a number of limitations with regards to how it detects certain tools. For JavaScript libraries and frameworks, the limitations are described in more detail <a href="https://github.com/AliasIO/wappalyzer/issues/2450">here.</a></aside>
```

Comment:
Rather than suggesting that the results are inaccurate, which I would dispute, I think all that needs to be acknowledged is that there's a known limitation or blind spot in the Wappalyzer library and these numbers should be taken as a lower bound.

All of our tools have blind spots: HA only measures home pages, CrUX excludes SPAs, Lighthouse only measures on mobile, WebPageTest only runs in Chrome, Wappalyzer only looks at global JS, etc. We acknowledge these limitations on the Methodology page but we haven't dissuaded readers from trusting the results.

Reply:
I decided to mention that because for this specific scenario, the blind spot may actually be quite large (which can affect the end result of "which framework is more popular than the rest").

But good point, there are many limitations with the tools used in the entire almanac. Removed the entire sentence 👍.